docker/run_mdswshell.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Sets up a shell in a container for building and debugging minidump-stackwalk.
+
+set -e
+
+INDOCKER=${INDOCKER:-0}
+
+if [[ ${INDOCKER} -eq 1 ]]; then
+    # Save the virtualenv in /app because that's mounted locally and saves some
+    # time between invocations
+    python -m venv /app/mdsw_venv/
+    source /app/mdsw_venv/bin/activate
+    pip install -r requirements/default.txt -c requirements/constraints.txt
```

Comment:
The socorro_breakpad intermediary docker image kicks off *before* it installs Python requirements. That's good for building the socorro_app image, but it sucks here because we want a shell in a container with all the things we need to build `minidump-stackwalk` **and** the Python requirements so we can use the scripts to download crash data and run `minidump-stackwalk` against it.

Given that this is a debugging tool, I decided to go for "gross, but it works". Because installing a lot of Python requirements every time you invoke `run_mdswshell.sh` stinks, I'm "saving that state" in a virtualenv. It's gross because now users have a virtualenv that's only used for mdswshell in their socorro root directory which make grepping and a bunch of other things work poorly. We can yak-shave on this some other day. My current prediction is that I will use the `mdswshell` and maybe two other people in the whole world, so gross is ok.